index.html

```diff
@@ -93,18 +93,20 @@ <h2 class="dd-wager-error-message">PLEASE ENTER A WAGER</h2>
       <button class="pop-up-buttons submit-dd-wager-button">SUBMIT WAGER</button>
     </section>
     <section class="hidden pop-ups final-round-pop-up">
-      <h1>FINAL ROUND</h1>
-      <h1 class="final-round-category">CATEGORY</h1>
+      <h1 class="final-category-text">FINAL ROUND</h1>
```

Comment:
Reviewed together, error msg looks good. Discussed having the pop-up cover full screen.